virtcontainers/qemu.go

```diff
@@ -643,6 +643,7 @@ func (q *qemu) stopSandbox() error {
 	defer span.Finish()
 
 	q.Logger().Info("Stopping Sandbox")
+	defer q.cleanupVmPath()
```

Comment:
Could you just ignore qmp errors and continue in the function? There is no need to do cleanup in a defer function.